core/shared/src/main/scala/org/http4s/util/Renderable.scala

```diff
@@ -114,6 +115,47 @@ object Renderer {
     new Renderer[A] {
       override def render(writer: Writer, t: A): writer.type = writer << select.toRaw1(t)
     }
+
+  val charRenderer: Renderer[Char] =
+    new Renderer[Char] {
+      override def render(writer: Writer, c: Char): writer.type =
+        writer << c
+    }
+
+  def stringLiteralRenderer(s: String): Renderer[Unit] =
+    new Renderer[Unit] {
+      override def render(writer: Writer, u: Unit): writer.type =
+        writer << s
+    }
+
+  def charLiteralRenderer(c: Char): Renderer[Unit] =
+    new Renderer[Unit] {
+      override def render(writer: Writer, u: Unit): writer.type =
+        writer << c
+    }
+
+  implicit val catsContravariantMonoidalForHttp4sUtilRenderable: ContravariantMonoidal[Renderer] =
```

Comment:
_tiny nitpick_: somewhere in http4s, we use implicit lazy vals. Shouldn't we apply it here also?